actions/class.RestQtiTests.php

```diff
@@ -161,6 +163,35 @@ public function importDeferred()
         }
     }
 
+    /**
+     * @throws common_exception_NotImplemented
+     */
+    public function getItems(): void
+    {
+        $request = $this->getPsrRequest();
+        $testUri = $request->getQueryParams()[self::PARAM_TEST_URI] ?? null;
+        try {
+            if ($request->getMethod() !== Request::HTTP_GET || empty($testUri)) {
+                throw new common_exception_MissingParameter(self::PARAM_TEST_URI, $this->getRequestURI());
+            }
+            $testResource = $this->getResource($testUri);
+
+            $this->returnSuccess(
+                array_map(static function (Resource $item) {
+                    return $item->getUri();
+                }, array_values($this->getQtiTestService()->getItems($testResource)))
+            );
+        } catch (MissingTestmodelException $e) {
+            $this->returnFailure(new common_exception_NotFound(
+                sprintf('Test %s not found', $testUri),
+                404,
```

Comment:
Could we have constant here?

Reply:
could you provide a bit more context here?

> Or fix link to SWAG

Reply:
Sorry I jump in test window during review it for previous comment

Reply:
added constant usage from `slim/slim`